src/lib/Libnet/net_server.c

```diff
@@ -472,6 +471,39 @@ engage_authentication(conn_t *pconn)
 	return (-1);
 }
 
+/*
+ * @brief
+ * process_socket  -  The static method processes given socket and
+ *                    engages the appropriate connection authentication.
+ *
+ * @param[in]   sock 	- socket fd to process
+ *
+ * @retval	-1 for failure
+ * @retval	0  for success
+ *
+ */
+static int
+process_socket(int sock)
```

Comment:
Better to keep the name of the function as process_priority_socket only.

Reply:
It is a common function to process all the fd's .

Reply:
Now that you are using this function for normal as well as priority sockets no need to rename it.

Reply:
done